playwright/support/helper.ts

```diff
@@ -61,6 +61,15 @@ export const checkAccessibility = async (
     .disableRules([...preDisabledRules, ...disableRules])
     .analyze();
 
+  if (accessibilityScanResults.incomplete.length > 0) {
```

Comment:
comment: I know we have plenty of lint warnings in master but I think it's worth adding a lint ignore here so we're not adding more etc